### PR TITLE
LPD-38808 Test using zulu17

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

**Comment by julschong** [comment]
ci:test:smoke

**Comment by julschong** [comment]
ci:test:smoke

File: test.properties [hunk 1]
```diff
@@ -9563,9 +9563,9 @@
     test.batch.dist.app.servers[smoke]=tomcat
 
     test.batch.names[smoke]=\
-        functional-smoke-tomcat90-mysql57
+        functional-smoke-tomcat-mysql57-jdk17_zulu
```
Comment (julschong) [inline test.properties:9566]: @vicnate5 is this jdk17_zulu syntax correct? seems to not take it and still used jdk8 on https://testray.liferay.com/#/project/35392/routines/45357/build/35860829

Comment (vicnate5) [reply test.properties:9566]: Please check with CI team

**Pull request closed without merging**